pkg/types/types.go

```diff
@@ -138,6 +138,9 @@ type BasicRuntimeMetadata struct {
 	// containerd: events from both initial and new containers are enriched
 	// crio: events from initial containers are enriched
 	ContainerImageDigest string `json:"containerImageDigest,omitempty" column:"containerImageDigest,hide"`
+
+	// ContainerStartedAt is the unix timestamp at which the container was started at
+	ContainerStartedAt int64 `json:"containerStartedAt,omitempty" column:"containerStartedAt,hide"`
```

Comment:
It should use the `types.Time` type so that the columns output shows human-readable time.

See also:
https://github.com/inspektor-gadget/inspektor-gadget/blob/main/pkg/types/types.go#L64-L70

Reply:
No problem. Although, I'm wondering if maybe we want to convert this to something human-readable only when we output columns. WDYT?

Reply:
I don't know but I think it should be consistent with other timestamps.

Any other opinions?